Project_source/app/src/main/java/android/clicker/school_live_simulator/MountainBicycleState.kt

```diff
@@ -0,0 +1,11 @@
+package android.clicker.school_live_simulator
+
+class MountainBicycleState : UsualBicycleState() {
+    override val price = 40000
+
+    init {
+        available_delivery.add(Delivery.FAST_DELIVERY)
```

Comment:
@Ferrociraptus is this line that you really wanted? To my mind, it is the same as available_delivery.add(5000) and it doesn't give us information about names of deliveries that are available, which we should use or am I mistaken?

Reply:
Andrey wrote the code correctly. 
Firstly, code is better to read. 
Secondly time it is the only true implementation because Kotlin statically typed lang and this list is the list of type Delivery.

Reply:
> Andrey wrote the code correctly.
> Firstly, code is better to read.
> Secondly time it is the only true implementation because Kotlin statically typed lang and this list is the list of type Delivery.

The question wasn't about the code the question was about the  **_line_**. I meant that if we did like that: _available_delivery.add(Delivery.FAST_DELIVERY)_ , would available_delivery look like [5000]? Actually, it looks like [FAST_DELIVERY] which is that we actually need(I asked because I wasn't sure about that. In Andrey's code everything is fine but you didn't answer my question. You wrote just something:) ). 
P.s. I also don't understand your phrase 'Secondly time' did you mean just 'Secondly'?

Reply:
> The question wasn't about the code, it was about the line. I meant that if we did like that: available_delivery.add(Delivery.FAST_DELIVERY) , would available_delivery look like [5000]? Actually, it looks like [FAST_DELIVERY] which is that we actually need(I asked because I wasn't sure about that. In Andrey's code everything is fine but you didn't answer my question.

@Bazalii, I think, it will contain Delivery class object, it means that available_delivery.add (Delivery.FAST_DELIVERY), would be available_delivery [<index>] look like FAST_DELIVERY and if you want to get salary you should be written available_delivery [<index>].salary.
This list is for checking object contains, but we both know that the best way for this operation is set, but in our case, it is not critical.


> P.s. I also don't understand your phrase 'Secondly time' did you mean just 'Secondly'?

Yes, thank you it is my mistake.

Reply:
> > The question wasn't about the code, it was about the line. I meant that if we did like that: available_delivery.add(Delivery.FAST_DELIVERY) , would available_delivery look like [5000]? Actually, it looks like [FAST_DELIVERY] which is that we actually need(I asked because I wasn't sure about that. In Andrey's code everything is fine but you didn't answer my question.
> 
> @Bazalii, I think, it will contain Delivery class object, it means that available_delivery.add (Delivery.FAST_DELIVERY), would be available_delivery [] look like FAST_DELIVERY and if you want to get salary you should be written available_delivery [].salary.
> This list is for checking object contains, but we both know that the best way for this operation is set, but in our case, it is not critical.
> 
> > P.s. I also don't understand your phrase 'Secondly time' did you mean just 'Secondly'?
> 
> Yes, thank you it is my mistake.

Yep, I also wrote that I understood that it would look like [FAST_DELIVERY] but thx for more commentaries!